shell/platform/darwin/ios/framework/Source/FlutterChannelKeyResponderTest.mm

```diff
@@ -18,9 +18,12 @@
 
 namespace {
 API_AVAILABLE(ios(13.4))
+// NOLINTNEXTLINE(readability-identifier-naming)
```

Comment:
Why do want to suppress this rather than fix the naming?

Reply:
Done. There is actually a constant for it, I removed this declaration.